pkg/sql/type_change.go

```diff
@@ -794,7 +794,7 @@ func (t *typeSchemaChanger) canRemoveEnumValueFromArrayUsages(
 				if !firstClause {
 					unionUnnests.WriteString(" UNION ")
 				}
-				unionUnnests.WriteString(fmt.Sprintf("SELECT unnest(%s) FROM [%d AS t]", col.GetName(), ID))
+				unionUnnests.WriteString(fmt.Sprintf("SELECT unnest(t.%s) FROM [%d AS t]", col.GetName(), ID))
```

Comment:
you also probably want to use `tree.Name(col.GetName())` to escape this properly too? (i think if you test a column names `bob""b` this will break)

Reply:
i believe `query.WriteString(fmt.Sprintf(" t.%s = %s", col.GetName(), sqlPhysRep)` has the same problem.

Reply:
ah you should use `ColName` for both of these.

Reply:
Can we generate these strings using tree formatting logic? The Name type will properly quote itself I think.